packages/backend/test/git.test.ts

```diff
@@ -0,0 +1,17 @@
+import fs from 'fs';
+import { getGitUrl, removeGitRepo } from '../src';
+
+jest.mock('fs');
+
+describe('git', () => {
```

Comment:
What do you think about adding a test for `cloneGitRepo` function?

Reply:
Done